### ISSUE TITLE
Connect isn't succesfull w/o IP address (and MAC)

### ISSUE THREAD

**Issue by JanneKiiskila** [ARMmbed/easy-connect#8]
We shouldn't return SUCCESS, if we do not get an IP address (and MAC).

```
    const char *ip_addr = network_handler->get_ip_address();
    const char *mac_addr = network_handler->get_mac_address();
    if (ip_addr && mac_addr) {
        pc.printf("IP address %s\r\n", ip_addr);
        pc.printf("MAC address %s\r\n", mac_addr);
    } else {
        pc.printf("[main] No IP/MAC address\r\n");
        return NULL;
   }
```

**Comment by JanneKiiskila** [comment]
This was also added via mbed-client.

**Comment by janjongboom** [comment]
Landed in #11.
